docs/general/builtins/scalar-functions.rst

```diff
@@ -3962,6 +3962,17 @@ Example::
     +----------+
     SELECT 1 row in set (... sec)
 
+.. NOTE::
+
+    For unknown schemas:
+
+    - Returns ``TRUE`` for superusers.
+
+    - For a user with ``DQL`` on cluster scope, returns ``TRUE`` if the
+      privilege type is ``USAGE``.
+
+    - For a user with ``DML`` on cluster scope, returns ``TRUE`` if the
+      privilege type is ``CREATE``.
```

Comment:
I would add one more entry, to say that it returns FALSE in other cases, just to make it 100% clear.

Reply:
Thank you added